docs/loading_environment_variables.md

```diff
@@ -0,0 +1,160 @@
+# Loading environment variables
+
+imgproxy can load environment variables from various sources such as:
+
+* [Local file](#local-file)
+* [AWS Secrets Manager](#aws-secrets-manager)
+* [AWS Systems Manager Parameter Store](#aws-systems-manager-parameter-store)
+* [Google Cloud Secret Manager](#google-cloud-secret-manager)
+
+## Local file
+
+You can create an [environment file](#environment-file-syntax) and configure imgproxy to read environment variables from it.
+
+* `IMGPROXY_ENV_LOCAL_FILE_PATH`: the path of the environmebt file to load
+
+## AWS Secrets Manager
+
+You can store the content of an [environment file](#environment-file-syntax) in AWS Secrets Manager secret and configure imgproxy to read environment variables from it.
```

Comment:
```suggestion
You can store the content of an [environment file](#environment-file-syntax) in AWS Secrets Manager secret and configure imgproxy to read environment variables from it.
```

something is wrong here. "AWS Secrets Manager secret" -- is the second "secret" supposed to be there? if so, a word is missing

Reply:
No, your correction seems correct to me. 
```suggestion
You can store the content of an [environment file](#environment-file-syntax) in an AWS Secrets Manager secret and configure imgproxy to read environment variables from it.
```

Reply:
could also be "as" a secret. whichever you think is better.